sksparse/cholmod.pyx

```diff
@@ -397,7 +440,8 @@ cdef class Factor:
         cholmod_factorize_p(&c_A, [beta, 0], NULL, 0,
                             self._factor, &self._common._common)
         if self._common._common.status == CHOLMOD_NOT_POSDEF:
-            raise CholmodError("Matrix is not positive definite")
+            raise CholmodNotPositiveDefiniteError("Matrix is not positive definite", self._factor.minor, self)
```

Comment:
I think there is no error message at all, just the status code indicating the error